src/lambdas/initiateAthenaQuery/startQueryExecution.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { startQueryExecution } from './startQueryExecution'
+import { mockClient } from 'aws-sdk-client-mock'
+import {
+  AthenaClient,
+  StartQueryExecutionCommand
+} from '@aws-sdk/client-athena'
+
+const athenaMock = mockClient(AthenaClient)
+
+describe('start Query execution', () => {
+  beforeEach(() => {
+    athenaMock.reset()
+  })
+
+  test('returns a QueryExecutionId if a query is successfully initiated', async () => {
+    athenaMock.on(StartQueryExecutionCommand).resolves({
+      QueryExecutionId: '123'
+    })
+
+    const result = await startQueryExecution({
+      sqlGenerated: true,
+      idParameters: ['test_parameter'],
+      sql: 'test sql'
+    })
+    expect(result).toEqual({
+      queryExecuted: true,
```

Comment:
You can use the `expect(athenaMock).toHaveReceivedCommandWith` with the relevant Command to assert that the right parameters were passed in. For an example, see `src/sharedServices/queue/sendSqsMessage.spec.ts`